login-update (#246)
medic/cht-docs: content/en/apps/examples/training.md

## Workflow Examples

### Remote Login by App

> When creating users, the admin has the option to send a user their credentials via SMS using a magic link. This generates a new, random and complex password with a 24-hour expiry. 

Reviewer: may be helpful to add a link to the "magic link" - not sure everyone will know what it is

Author: good call!

> By clicking the magic link to log in, the user is able to bypass having to enter their username and password as it allows them to enter their project's instance directly. If the app is not installed on their phone, the default SMS encourages users to download it. 

Reviewer: ```suggestion
By clicking the magic link to log in, the user is able to enter their project's instance directly, bypassing the need to enter their username and password. If the app is not installed on their phone, the default SMS encourages users to download it. 
```
